src/modules/layouts/components/WebsiteLayout/Footer/Footer.jsx

```diff
@@ -39,29 +40,27 @@ const Footer = () => {
     [],
   );
 
+  const { error, response, submitForm } = useHubspotForm({
```

Comment:
:ok_hand: Not necessary to extract values but `formGuid` (x2) and `portalId` (x3) are being reused.